doc/operator-notes.md

```diff
@@ -9,3 +9,7 @@ For a given retry attempt, Ignition will wait 10 seconds for the server to send
 Any HTTP response code less than 500 results in the request being completed, and either the resource will be fetched or Ignition will fail.
 
 Ignition will initially wait 100 milliseconds between failed attempts, and the amount of time to wait doubles for each failed attempt until it reaches 5 seconds.
+
+## Ignition and IAM roles
```

Comment:
Curses, I made that `EC2 and IAM roles` when originally making this change but fat-fingered the git commands and had to remake it. Thoughts on that name, since this feature is specific to the EC2 platform?